charts/internal/machineclass/templates/machineclass.yaml

```diff
@@ -6,8 +6,7 @@ metadata:
   name: {{ $machineClass.name }}
   namespace: {{ $.Release.Namespace }}
 {{- if $machineClass.labels }}
-  labels:
-{{ toYaml $machineClass.labels | indent 4 }}
+  labels: {{- toYaml $machineClass.labels | nindent 4 }}
```

Comment:
I like the changes introducing consistent template directives' indentation and fixing possible indentation flaws.